### PR TITLE
Small fixes and cleanup

### PR DESCRIPTION
**Pull request by WhatisRT**

### CODE REVIEW

File: CIP-1694/README.md [hunk 1]
```diff
@@ -898,24 +887,22 @@ Regardless of any governance mechanism, SPO participation is needed for any hard
...
+The DReps also vote, to represent the will of every stake holder.
 
 ### New Metadata structures
 
 Both the governance actions and the votes use new metadata fields,
 in the form of URLs and integrity hashes
 (mirroring the metadata structure for stake pool registration).
 The metadata is used to provide context.
-For example, votes by the constitutional committee need an explanation as to why they are in line with
-the constitution.
```
Comment (JaredCorduan) [inline CIP-1694/README.md:910]: why did you remove this?

Comment (WhatisRT) [reply CIP-1694/README.md:910]: Votes currently don't have anchors. So we'd have to add them if we wanted this.

Comment (WhatisRT) [reply CIP-1694/README.md:910]: Oh, looks like I'm wrong. I thought we agreed that votes wouldn't have anchors, but it seems like it's still there.

Comment (JaredCorduan) [reply CIP-1694/README.md:910]: I see. Maybe we need to add an open question about votes having optional anchors. Or perhaps tx metadata would work better in this case... :thinking:

Comment (WhatisRT) [reply CIP-1694/README.md:910]: Oh, right, I think that's why we said that we wouldn't want anchors on there. I think adding that to the options at the bottom seems like a good idea

Comment (JaredCorduan) [reply CIP-1694/README.md:910]: :100:

File: CIP-1694/README.md [hunk 2]
```diff
@@ -494,16 +497,23 @@ will be dropped if the motion is enacted.
 #### Ratification
 
 Governance actions are **ratified** through on-chain voting actions.
-Different kinds of governance actions have different ratification requirements but always involve **two of the three** governance bodies.
+Different kinds of governance actions have different ratification requirements but always involve **two of the three** governance bodies,
+with the exception of a hard-fork initiation, which requires ratification by all governance bodies.
```
Comment (JaredCorduan) [inline CIP-1694/README.md:501]: Reminds me of that "twin riddle":

> Q: Two kids were born to the same mother, on the same day, at the same time, in the same month and year and yet they’re not twins. How is this possible?

> A: They are part of a set of triplets.

:grin:
